$NetBSD: patch-ap,v 1.1 2010/03/08 01:21:25 dholland Exp $

Don't declare own errno, it doesn't work on various platforms.

--- src/file/file.h~	2000-07-30 19:39:20.000000000 -0400
+++ src/file/file.h	2010-03-07 20:09:46.000000000 -0500
@@ -101,8 +101,11 @@ extern void  ckfprintf		__P((FILE *, con
 extern unsigned long signextend	__P((struct magic *, unsigned long));
 
 
-
+#if 0
 extern int errno;		/* Some unixes don't define this..	*/
+#else
+#include <errno.h>
+#endif
 
 extern char *progname;		/* the program name 			*/
 extern char *magicfile;		/* name of the magic file		*/
